pkg/modules/authorization.go

```diff
@@ -77,6 +77,8 @@ const (
 	AuthRedisCommanderImage = "<AUTHORIZATION_REDIS_COMMANDER_IMAGE>"
 	// AuthRedisStorageClass -
 	AuthRedisStorageClass = "<REDIS_STORAGE_CLASS>"
```

Comment:
We can probably remove references to redis storage classes as we dont use them anymore. This might not be the only place, maybe a different PR to clean it up would be better.